File: Inverse a number/code.java
import java.util.*;

public class Main{

 public static void main(String[] args) {
  // write your code here  
    Scanner scn = new Scanner(System.in);
    int n = scn.nextInt();

    int inverse = 0;
    int index = 1;
    while(n != 0){
        int last_digit = n % 10;
        int inverse_digit = index;
        int inverse_index = last_digit;

        inverse = inverse + index * (int)Math.pow(10,last_digit-1);

        index++;
        n = n/10;
    }

    System.out.println(inverse);
 }
}

File: Inverse a number/code.py
def inverse(n):
    # write your code here
    inverse = 0
    index = 1
    while n != 0:
        last_digit = n % 10
        inverse_digit = index
        inverse_index = last_digit

        inverse = inverse + index * pow(10,last_digit-1)

        index += 1
        n = n//10

    print(inverse)



def main():
    n = int(input())
    inverse(n)

if __name__=="__main__":
    main()